### PR TITLE
60th episode

### PR DESCRIPTION
**Pull request by maxandersen**
** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc **

### CODE REVIEW

**Review by phillip-kruger — APPROVED** [review]

File: _posts/2021-08-20-60th-quarkus-insights.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: Season 3 and 60th Insights episode
+date: 2021-08-20   
+tags: insights
+synopsis: Quarkus Insights Sesaon 3 starting!
```
Comment (gsmet) [inline _posts/2021-08-20-60th-quarkus-insights.adoc:6]:
```suggestion
synopsis: Quarkus Insights Season 3 starting!
```

File: _posts/2021-08-20-60th-quarkus-insights.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+author: maxandersen
+---
+
+:imagesdir: /assets/images/posts
+
+After a summuer break and little bit of CODVID-19 delay, we will finally have the 60th(!) Quarkus Insights episode on Monday the 23rd August.
```
Comment (gsmet) [inline _posts/2021-08-20-60th-quarkus-insights.adoc:12]:
```suggestion
After a summer break and little bit of COVID-19 delay, we will finally have the 60th(!) Quarkus Insights episode on Monday the 23rd August.
```

File: _posts/2021-08-20-60th-quarkus-insights.adoc [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+image:60th.png[]
+
+For those who don't know Quarkus Insights is a (almost) weekly video/audio podcast where we host people from all parts of the Quarkiverse to sit down and talk about how to use, what is coming next, user stories from the actual users and more!
```
Comment (gsmet) [inline _posts/2021-08-20-60th-quarkus-insights.adoc:16]:
```suggestion
For those who don't know, Quarkus Insights is a (almost) weekly video/audio podcast where we host people from all parts of the Quarkiverse to sit down and talk about how to use Quarkus, what is coming next, user stories from the actual users and more!
```

File: _posts/2021-08-20-60th-quarkus-insights.adoc [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+This monday is titled "The Rise of Platforms" where Alexey will come by and talk about the improvements and changes we've done around the notion of Quarkus Platforms and Extensions for Quarkus 2.x.
```
Comment (gsmet) [inline _posts/2021-08-20-60th-quarkus-insights.adoc:18]:
```suggestion
This Monday is titled "The Rise of Platforms" where Alexey will come by and talk about the improvements and changes we've done around the notion of Quarkus Platforms and Extensions for Quarkus 2.x.
```

File: _posts/2021-08-20-60th-quarkus-insights.adoc [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+We have a whole backlog of 60 episodes which for large part are still as relevant as they were when we recorded them.
+
+You can browse and see the full list at http://quarkus.io/insights/
```
Comment (gsmet) [inline _posts/2021-08-20-60th-quarkus-insights.adoc:22]:
```suggestion
You can browse and see the full list at http://quarkus.io/insights/.
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Spotted a couple of typos.

**Pull request merged**